src/components/DropdownItem/DropdownItem.tsx

```diff
@@ -0,0 +1,41 @@
+import cn from 'classnames';
+import { Person } from '../../types/Person';
+
+type Props = {
+  person: Person;
+  onSelectedPerson: (person: Person) => void;
+  onQuerry: (value: string) => void;
+};
+
+export const DropdownItem: React.FC<Props> = (
+  {
+    person,
+    onSelectedPerson,
+    onQuerry,
+  },
+) => {
+  const handleSelectedPersone = (
+    event: React.MouseEvent<HTMLAnchorElement, MouseEvent>,
+  ) => {
+    event?.preventDefault();
+    onSelectedPerson(person);
+    onQuerry(person.name);
+  };
+
+  return (
+    <a
+      key={person.slug}
```

Comment:
The 'key' prop is unnecessary here. It is used in the parent component when rendering a list of items, not within the component itself.